### PR TITLE
Plague touch big buff

### PR DESCRIPTION
**Pull request by MatheusIsNotAvailable**
[BALANCE] [WIKI] [INPUT]

## About the PR 
Plague touch now does more damage, more often, costs less, is unlocked earlier and has a lower cooldown 
## Why's this needed? 
Grave touch is a very underwhelming ability, it giving people a disease that is at it's worst annoying simply won't do, vampire is also a very weak antag. 
## Changelog

```changelog
(u)CodeDude
(*)The vampire ability, plague touch has gotten a big buff, it has a lower cooldown(60), you need less blood to unlock it(900), it costs less(30) and it also is way more lethal

### CODE REVIEW

**Comment by Momoberry** [comment]
Definitely think this is a good thematic addition, cause grave touch barely works currently, takes like 25 minutes to even be lethal, and has no real interaction with the person, similar to sleepy pen. It's more just like giving people cooties.

 This can hopefully make vamp a bit more aggressive and not afraid of a staffie with a welder, even if its just to throw em away or something.

**Review by Tarmunora — CHANGES_REQUESTED** [review]
either change from chokehold to aggressive grab in code, or change messaging/descriptions/etc to reflect that it's a chokehold and not an aggressive grab

**Review by Tarmunora — APPROVED** [review]

**Comment by MatheusIsNotAvailable** [comment]
I have been thinking about raising it's blood cost to something like 100 and lowering it's cooldown a little. To make it an ability you have to follow up on as you will probably be drinking blood off of the person you use it on.

 Also been thinking about making the disease cause slowdown on people on all of it's 3 stages(it only does this on it's 3rd stage currently) and lowering the damage a little. So it is harder to run away after it gets used on you. What do yall think of this?

**Review by flrsh — CHANGES_REQUESTED** [review]
Mbc and I talked about this, and we think that this brings vamp closer to the niche of ling (with a guaranteed "I get you" ability). I think instead of replacing plague touch with this new ability, you should just buff plague touch.

**Comment by Momoberry** [comment]
Just raw stat changes? Won't that just fall into the entire issue pathology had and why it's literally no longer in the game currently? I think the ability itself should just be reworked. If not the grasp idea, then keep the unlock cost at like 2000 blood, make it cost 750 to use and have it turn somebody into a vampire over five minutes with various flavour text messages that are about feeling urges for blood, etc. Idk. 

The reason friendly vampires are common is because there's no danger to letting a vamp have blood beyond 600 cause of ice bats. None of the later abilities are even moderately threatening compared to frost bats, including how grave touch is. And if you buff it so it's strong, it's gonna have the same issue ling sting has where theres no interaction for the victim of the touch and either they meta knowledge purge it or they probably die.

**Comment by MatheusIsNotAvailable** [comment]
I dunno, I don't think turning people into vampires with an ability is good, people would just either ask to be turned or vampires would offer themselves to turn people into vampires too wich in both cases is lame, I am giving it some time to see if I can think of anything special for the ability to do, I don't think just changing the stats would be enough to make it ok, If I can't think of anything I will just change the stats.

**Comment by flrsh** [comment]
Thanks! Can you also lag adjust it (use the mult thing) to compensate for changes in the tickrate of the life loop?

**Review by flrsh — APPROVED** [review]
Sorry I overlooked this!

File: code/modules/medical/diseases/vamplague.dm [hunk 1]
```diff
@@ -8,27 +8,28 @@
...
+			affected_mob.changeStatus("slowed", 3 SECONDS * mult)
 		affected_mob.take_toxin_damage(toxdamage)
 
-	if (prob(5))
+	if (probmult(10))
 		boutput(affected_mob, "<span class='alert'>Your joints ache horribly!</span>")
-		affected_mob.changeStatus("weakened", stuntime SECONDS)
-		affected_mob.changeStatus("stunned", stuntime SECONDS)
+		affected_mob.changeStatus("weakened", stuntime SECONDS * mult)
+		affected_mob.changeStatus("stunned", stuntime SECONDS * mult)
```
Comment (Tarmunora) [inline code/modules/medical/diseases/vamplague.dm:31]: double-compensating

File: code/modules/medical/diseases/vamplague.dm [hunk 2]
```diff
@@ -8,27 +8,28 @@
...
 		affected_mob.emote(pick("cough","groan", "gasp"))
 		affected_mob.losebreath++
 
-	if (prob(15))
-		if (prob(33))
+	if (probmult(15))
+		if (probmult(33))
 			boutput(affected_mob, "<span class='alert'>You feel sickly and weak.</span>")
-			affected_mob.changeStatus("slowed", 3 SECONDS)
+			affected_mob.changeStatus("slowed", 3 SECONDS * mult)
```
Comment (Tarmunora) [inline code/modules/medical/diseases/vamplague.dm:25]: doublecompensating

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/modules/medical/diseases/vamplague.dm [hunk 3]
```diff
@@ -8,27 +8,28 @@
...
 
-	if (prob(10))
+	if (probmult(10))
 		affected_mob.emote(pick("cough","groan", "gasp"))
 		affected_mob.losebreath++
 
-	if (prob(15))
-		if (prob(33))
+	if (probmult(15))
+		if (probmult(33))
```
Comment (Tarmunora) [inline code/modules/medical/diseases/vamplague.dm:23]:
```suggestion
		if (prob(33))
```

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**